ui/app/templates/components/raft-storage-restore.hbs

```diff
@@ -0,0 +1,70 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="key-value-header breadcrumb">
+      <ul>
+        <li>
+          <span class="sep">&#x0002f;</span>
+          {{#link-to "vault.cluster.storage"}}
+            Raft Storage
+          {{/link-to}}
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      Restore Snapshot
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<form {{action (perform this.restore this.file) on="submit"}}>
+  <MessageError @errors={{this.errors}} />
+
+  {{#if this.isUploading}}
+    <div class="box is-sideless is-fullwidth is-marginless">
+    <AlertBanner
+      @type="warning"
+      @title="Uploading your file..."
+      @message="Raft snapshots can be very large files. Be prepared for uploading the file to take some time."
```

Comment:
grammar nit, but i think this message could just be "Raft snapshots can be very large files. Uploading the snapshot may take some time."